### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by aditikum**
### Description
Updating readme to use terminology of ASP.

### CODE REVIEW

File: n6001/README.md [hunk 1]
```diff
@@ -50,13 +50,13 @@ and updates the project qsf files appropriately.
 
 Need to set **OPAE_PLATFORM_ROOT** to point to ofs-dev/work_ofs_ac_base_adp-A1/pr_build_template in FIM build area.
 
-Need to set **OFS_OCL_SHIM_ROOT** to point to opencl-bsp/n6001.
+Need to set **OFS_OCL_SHIM_ROOT** to point to opencl-asp/n6001.
```
Comment (pajgaonk) [inline n6001/README.md:53]: its should be oneapi-asp

Comment (pajgaonk) [reply n6001/README.md:53]: we have also changed OFS_OCL_SHIM_ROOT to OFS_ASP_ROOT. I think most of change sin this PR might already be in master now

Comment (aditikum) [reply n6001/README.md:53]: good catch

Comment (aditikum) [reply n6001/README.md]: found it in d5005 fixing that as well

Comment (aditikum) [reply n6001/README.md]: > we have also changed OFS_OCL_SHIM_ROOT to OFS_ASP_ROOT. I think most of change sin this PR might already be in master now

Synced to main and then saw these!

**Review by pajgaonk — APPROVED** [review]
Looks good Thank you

**Pull request merged**